src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+// Register means every single file will get processed by babel that we import!
+import 'babel/register'
+
+import yaml from 'js-yaml';
+import fsPath from 'path';
+import fs from 'mz/fs';
+import createCommands from './commands';
+
+const CONFIG = 'exhibition.yml';
```

Comment:
No, I was just suggestion a cli wrapper thingy that launches the bash script above...by walking up the directory tree and launch the first bash script called `exhibition` so people don't have to add the bash script the their `PATH`.

It's a bit weird with mach to have to add something from a repo to your PATH.